z3tracer/src/parser.rs

```diff
@@ -212,7 +212,7 @@ where
             }
             "[instance]" => {
                 let key = lexer.read_key()?;
-                let term = lexer.read_ident()?;
+                let term = lexer.read_optional_ident()?;
```

Comment:
I suppose this is part of supporting proof=false? Does it mean that when proof=false, it's possible to generate `[instance]` logs without terms? Why is that the case?

Reply:
No sure if it is "intended" that we don't have this term when proof=false but this shouldn't be an issue.